qt/scientific_interfaces/ISISReflectometry/Reduction/PolarizationCorrections.h

```diff
@@ -45,35 +37,25 @@ polarizationCorrectionTypeToString(PolarizationCorrectionType correctionType) {
 
 inline bool polarizationCorrectionRequiresInputs(
     PolarizationCorrectionType correctionType) {
-  return (correctionType == PolarizationCorrectionType::PA ||
-          correctionType == PolarizationCorrectionType::PNR);
+  UNUSED_ARG(correctionType);
```

Comment:
I can't find this function being used called anywhere. maybe it should be removed? Same with the `polarizationCorrectionTypeToString` above this one.

Reply:
I think I missed that - I'll remove it :+1: